src/openrct2/ride/coaster/JuniorRollerCoaster.cpp

```diff
@@ -1841,11 +1844,18 @@ static constexpr const uint32_t junior_rc_track_pieces_diag_blockbrakes[2][4] =
     },
 };
 
-void JuniorRCPaintTrackFlat(
-    PaintSession& session, const Ride& ride, uint8_t trackSequence, uint8_t direction, uint16_t height,
-    const TrackElement& trackElement, JuniorRCChainType chainType)
+template<JuniorRCSubType subType> constexpr uint8_t JuniorRCGetSubTypeOffset(const TrackElement& trackElement)
```

Comment:
I think we use typically T prefix for template parameters, so TSubType.

Reply:
Is `TJuniorRCSubType` also okay? (It’s a parameter specific to the Junior/Water Coaster track.)

Reply:
A bit long for my taste and its a strong type but I'm not against it.